torch/_functorch/partitioners.py

```diff
@@ -863,6 +863,9 @@ def is_fusible(a, b):
             return True
         if can_fuse_into_triton_kernel_wrapper_functional(a, b):
             return True
+        if a.target is operator.getitem and a.args[0].target is torch.ops.higher_order.triton_kernel_wrapper_functional:
+            # if a is the output of a user triton kernel, then (by default) we will not be able to fuse b into it
+            return False
```

Comment:
Can you write a test?

Separately, what did the failure mode look like?

Reply:
test added, also updated the description.

The failure mode was that the compiled forward from inductor contained 2 kernels (`user_triton`, `dedicated_inductor_kernel_for_sigmoid`), when the "better" case would have been to move the sigmoid() to the backward (so it could be fused into an existing inductor kernel in the backward)